beetsplug/lyrics.py

```diff
@@ -659,26 +671,133 @@ def commands(self):
             help=u'print lyrics to console',
         )
         cmd.parser.add_option(
+            u'-r', u'--write-rst', dest='writerst',
+            action='store', default='.', metavar='directory',
+            help=u'write lyrics to given directory as RST files',
```

Comment:
I know this is totally bikeshedding, but reStructuredText is usually abbreviated reST or ReST rather than RST.